README.md

```diff
@@ -2,15 +2,28 @@
 
 [![CircleCI Build Status](https://circleci.com/gh/1Password/secrets-orb.svg?style=shield "CircleCI Build Status")](https://circleci.com/gh/1Password/secrets-orb) [![CircleCI Orb Version](https://badges.circleci.com/orbs/onepassword/secrets.svg)](https://circleci.com/orbs/registry/orb/onepassword/secrets) [![GitHub License](https://img.shields.io/badge/license-MIT-lightgrey.svg)](https://raw.githubusercontent.com/1Password/secrets-orb/main/LICENSE) [![CircleCI Community](https://img.shields.io/badge/community-CircleCI%20Discuss-343434.svg)](https://discuss.circleci.com/c/ecosystem/orbs)
 
-The 1Password Secrets CircleCI Orb allows loading secrets from 1Password into CircleCI CI/CD pipelines, and syncing them automatically, therefore, removing the risk of exposing sensitive values.
+With 1Password Secrets orb for CircleCI, you can load secrets from 1Password into CircleCI CI/CD pipelines and sync them automatically. Using this orb removes the risk of exposing plaintext secrets in code.
+
 This orb is officially supported and maintained by 1Password, but community contributions are welcome.
 
-## Usage
+Read more on the [1Password Developer Portal](https://developer.1password.com/ci-cd/circle-ci). 
+
+## Requirements
+
+Before you get started, you'll need to:
+
+- Sign up for 1Password ([Get 50% off your first year](https://start.1password.com/sign-up?c=DEV2022)).
```

Comment:
I think we only mention this part in the docs portal, right? I don't think it should be included in the README.
```suggestion
```

Reply:
Good to know. This change seems fine to me and I'll keep it in mind going forward! :)